pyproject.toml

```
@@ -56,6 +56,9 @@ repository = "https://github.com/jazzband/pip-tools"
 changelog = "https://github.com/jazzband/pip-tools/releases"
 
 [project.optional-dependencies]
+supported-pip = [
+  "pip < 25.3",
```

Since #2253 merged, this should probably be:

```suggestion
  "pip <= 25.3",
```

---

@webknjaz This thread has been resolved, but the suggestion had not been applied, and the version dependency has not been changed in any other way.